prometheus/auto.d/ProductionOSWin.yml.example

```diff
@@ -0,0 +1,4 @@
+- targets: [ "10.123.202.12:9100" ]
```

Comment:
May be using wmi will not show systems on OS overview.
Either use "node" or change "node" to "os"

Reply:
Can you change the filter for the dasboard to look for `node` or `wmi`?

Reply:
does this have to match the name of the exporter or is it arbitrary? I thought it had to match, hence my changing it. i can easily set it back to `node` if its just an arbitrary label/filter. do either of you know for sure?

Reply:
actually, i see this is just a label. changing it back to `node`

Reply:
It's arbitrary. It's something that Yogesh added to make the filtering of the different metrics possible in Grafana. I think if we can change the filters in Grafana to use an `or` condition for both of them, it's probably better to leave this labeled as `wmi` so we can keep the different exporter types separated if needed